sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -333,12 +337,27 @@
     "xgb.fit({\"train\": train_input}, logs=False)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"DEMO-clarify-model-{}\".format(datetime.now().strftime(\"%d-%m-%Y-%H-%M-%S\"))\n",
+    "try:\n",
+    "    session.delete_model(model_name)\n",
+    "except:\n",
+    "    pass\n",
+    "model = xgb.create_model(name=model_name)\n",
+    "container_def = model.prepare_container_def()\n",
+    "session.create_model(model_name, role, container_def)"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Create Model\n",
-    "Here we create the SageMaker model."
+    "We can use the model above as an input to Clarify, in this case Clarify will spin up an endpoint for the duration of the job. In order to iterate faster it can be beneficial to use an already existing endpoint, we will create our own enpoint below."
```

Comment:
nit: "endpoint" is misspelled in the last sentence.

Reply:
fixed